lib/helpers/remote-config.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:firebase_remote_config/firebase_remote_config.dart';
+import '../global.dart';
+
+Future<dynamic> getApplicationConfiguration(String key) async {
+  final RemoteConfig remoteConfig = await RemoteConfig.instance;
+
+  try {
+    await remoteConfig.setDefaults(remoteCofigDefaults);
+    await remoteConfig.fetch(expiration: const Duration(seconds: 5));
+    await remoteConfig.activateFetched();
+    dynamic value =
+        remoteConfig.getString(key).trim();
+    return value;
```

Comment:
This function is doing a lot of work. It's initializing the config and also getting the required value from config. Can we split it into two functions?

Reply:
Example: https://github.com/FirebaseExtended/flutterfire/blob/master/packages/firebase_remote_config/example/lib/main.dart#L56